backend/config.py

```diff
@@ -17,15 +17,18 @@ class DevelopmentConfig(Config):
     TESTING = True
 
     """db config"""
-    SQLALCHEMY_DATABASE_URI = "postgresql://{username}:{password}@{hostname}:{port}/{databasename}".format(
-        username=os.environ.get("DB_USER"),
-        password=os.environ.get("DB_PASSWORD"),
-        hostname=os.environ.get("DB_HOST"),
-        port=5432,
-        databasename=os.environ.get("DB_NAME"),
+    DB_USER = os.environ.get("DB_USER")
+    DB_PASSWORD = os.environ.get("DB_PASSWORD")
+    DB_HOSTNAME = os.environ.get("DB_HOST")
+    DB_PORT = 5432
+    DB_NAME = os.environ.get("DB_NAME")
+
+    SQLALCHEMY_DATABASE_URI = (
+        f"postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOSTNAME}:{DB_PORT}/{DB_NAME}"
     )
     SQLALCHEMY_ENGINE_OPTIONS = {"pool_recycle": 299}
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     SECRET_KEY = os.environ.get("SECRET_KEY")
 
     JWT_BLACKLIST_ENABLED = True
+    JWT_ERROR_MESSAGE_KEY = "msg"
```

Comment:
Is this required? From documentation `flask-jwt-extended`:

> Defaults to 'msg'.

Is there any reason?